packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -263,6 +263,11 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
+   * An optional target when using href
+   */
+  target?: '_blank' | '_self' | '_parent' | '_top';
```

Comment:
Why not allow for arbitrary targets, so the same link can open in the same window every time?